data/quarg missions.txt

```diff
@@ -111,3 +111,75 @@ mission "First Contact: Kuwaru Efreti"
+			`	A pit forms in your stomach...maybe this was a bad idea.` 
+			choice 
+				`	(There's no going back now. Keep scanning...look for the best way to slice open the hull.)`
+					goto hull
+				`	(Maybe it would be easier to capture the ship than to rip it apart. Try scanning the inside of the ship for defensive measures.)`
+					goto capture
+				`	(Abandon this insane project. No alien technology is worth your life.)`
+					# defer the mission to be tried again later and boarded ship is destroyed to prevent re-boarding.
+					goto selfdestruct 
+
+			label capture
+			`	You tune your sensors to inspect the living areas of the enemy ship, starting at what you're fairly certain is the airlock. It doesn't take long for your heart to sink at what you're seeing. There are literally hundreds of tiny, independent power sources distributed over the ships' internal surfaces. They appear to power some kind of intrusion countermeasures. Maybe with a thousand marines you would have a prayer of capturing this ship. Maybe.`
+			choice 
+				`	(Ok, keep scanning. Look for a way to open the hull.)`
+					goto hull
+			
+			label hull
+			`	You continue scanning the Quarg ship's hull, looking for the best place to cut it open. There's already significant weapon damage - disabling the ship wasn't easy. You identify a few components: engines, what you think is the shield generator, and some kind of...battery? You activate your salvage cutter. It takes an agonizingly long time; the material is like nothing you've ever tried cutting through before.  You eventually succeed in opening some access holes, only to find that the components are locked in place with some king of energy anchors.  You can't figure out how to cut their power.`
+			`	Suddenly your thoughts are interrupted by a flashing light on your ship's comm. It looks like the Quarg want to talk.`
+			choice
+				`	(Ignore the comm and focus on taking this ship apart.)`
+					goto end
+				`	(Let's hear what they have to say.)`
+					goto listen
+
+			label listen
+			`	You see the image of a Quarg on your view screen, its alien features revealing no hint of its emotions- it's mannerisms seem the same as every other Quarg you've seen. You can only assume this is the captain of the ship.`
+			`	"Human," it addresses you in a measured tone, "Be warned that you are making a serious error. This one understands that it is only natural for a creature such as you to test itself in battle. However, for one such as you to possess Quarg tools or the Drak's gift is taboo. It can only lead to destruction. Your power has already been shown. Withdraw now and leave this system. Then peace will remain between us."`
```

Comment:
It can only lead to destruction - A species as young as yours cannot posess technology this advanced without creating a potentally disasterous power ripple in the galatic hiearchy. (A lot of mispellings, I know, but I think I got the genreal idea though)